File: 1897 redistribute-characters-to-make-all-strings-equal/solution.java
class Solution {
    public boolean makeEqual(String[] words) {
        if(words.length==1) return true;
        int count=0;
        for(String s:words) count+=s.length();
        if(count%words.length!=0) return false;
         int[] freq = new int[26];
        for(String s:words) {
            for(char c:s.toCharArray()) freq[c-'a']++;
        }
        for(int i=0;i<26;i++){
            if(freq[i]%words.length!=0) return false;
        }
        return true;
    }
}

File: 1897 redistribute-characters-to-make-all-strings-equal/solution.go
func makeEqual(words []string) bool {
    f:=make([]int,26)
    for _,v := range words {
        for _,c := range v {
            f[c-'a']++
        }
    }
    val:=len(words)
    for _,v := range f {
        if v%val !=0 {
            return false
        }
    }
    return true
}
